tests/compile-fail/invalid_upcast_comparisons.rs

```diff
@@ -0,0 +1,19 @@
+#![feature(plugin)]
+#![plugin(clippy)]
+
+#![deny(invalid_upcast_comparisons)]
+#![allow(unused, eq_op, no_effect)]
+fn main() {
+    let zero: u32 = 0;
+    let u8_max: u8 = 255;
+
+    (u8_max as u32) > 300; //~ERROR Because of the numeric bounds prior to casting, this expression is always false.
```

Comment:
How would I go about getting the string `u8_max` in that situation? Is there a utility function somewhere that gives the string representation of an expression?